content/common/contributing-docs.md

```diff
@@ -233,15 +238,63 @@ npx markdownlint-cli2 content/spin/install.md \"#node_modules\"
+
+**Word Mismatch**
+
+The words in your markdown file (that may help a user) may not be words that a user is actually searching for. For example, you may write about "different HTTP listening options" whereas a user may only ever try to find that specific content using a search phrase like "request alternate port". In this case (if/when you are aware of user search patterns), it is recommended that you reword your content to include the potential user's search phrase. If this is not possible (there is a mismatch between words in your content and works a user searches for) then you can alternatively use the `keywords` string in the `[extra]` section of your document's frontmatter. For example, the following show frontmatter that helps a user find your page (when using `port` in their search phrase):
+
+```markdown
+[extra]
+keywords = "port ports"
+```
+
+Adding a word to the `keywords` string of a page overrides the in-house search functionality by at least one order of magnitude. Adding a word to the `keywords` string may displace other content, so please use it only if necessary.
+
+The `keywords` string is designed to draw users to the start of a page. In some cases, this is not ideal. You may want to hone in on specific content to resolve a search query.
+
+**Honing in on specific content**
+
+As we just mentioned the in-house search works most of the time. However, if you know that a user will be searching for a specific phrase and you want to hone in on specific content (when a user clicks a search result) you may use the following syntax; anywhere in the body of your markdown file and the user's search action will direct them straight to the previous heading (nearest heading above the `@searchTerm` syntax).
+
+```markdown
+<!-- @searchTerm "honing" -->
+```
+
+<!-- @searchTerm "honing" -->
+
+When using the above `@searchTerm` feature, please note the following:
+- the words must be separated by a space.
+- these keywords will be boosted in the search results by at least one order of magnitude; so please use them with caution, so as not to displace other valid pages containing similar content.
+
+Example: If you search for the word "honing", the results will point you to the previous heading in this specific section of the developer documentation.
+
+![honing example](/static/image/honing.png)
+
+### 6.4 The Edit On GitHub Button
+
+Each markdown file in the developer documentation requires a link to its GitHub page for the site to correctly render the "Edit on GitHub" button for that page.
+
+![edit on github](/static/image/edit-on-github.png)
+
+If you create a new markdown file and/or you notice a file without the explicit GitHub URL please attend to this. For example, the following `url` is required for this page that you are reading (you can check the raw markdown contents to see this example):
```

Comment:
"Please attend to this" has, to me, a peremptory and bureaucratic tone.  Could we change it for something like "please add a `url` entry to the `[extra]` section" maybe?

Reply:
Noted and updated. Thanks.